Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Component/ComponentApplicationBus.h>
+#include <AzCore/DOM/DomPath.h>
+#include <AzCore/DOM/DomUtils.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/Name/Name.h>
+#include <AzCore/RTTI/AttributeReader.h>
+#include <AzCore/RTTI/TypeInfo.h>
+#include <AzFramework/DocumentPropertyEditor/PropertyEditorSystemInterface.h>
+#include <AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.h>
+
+namespace AZ::Reflection
+{
+    namespace DescriptorAttributes
+    {
+        Name Handler = Name::FromStringLiteral("Handler");
+        Name Label = Name::FromStringLiteral("Label");
+        Name SerializedPath = Name::FromStringLiteral("SerializedPath");
+    } // namespace DescriptorAttributes
+
+    namespace LegacyReflectionInternal
+    {
+        struct InstanceVisitor
+            : IObjectAccess
+            , IAttributes
+        {
+            IReadWrite* m_visitor;
+            SerializeContext* m_serializeContext;
+
+            struct AttributeData
+            {
+                Name m_group;
+                Name m_name;
+                Dom::Value m_value;
```

Comment:
Some comments here would be good. I am a bit unclear about what m_group here is used for.